scripts/setup_imdb.py

```diff
@@ -37,7 +37,7 @@ def is_setup():
 
 # This file contains the IMDB dataset and is based on the specifications of the
 # original JOB repository: https://github.com/gregrahn/join-order-benchmark
-LOCATION = "http://www.dropbox.com/s/fgju7cifpehqlww/imdb.zip?dl=1"
+LOCATION = "https://www.dropbox.com/s/45y07uh4chjy8ie/imdb_data.zip?dl=1"
 FILE_NAME = "imdb.zip"
```

Comment:
Since the file name has changed to `imdb_data.zip`, do we need to adjust the following line as well?

```bash
FILE_NAME = "imdb.zip"
```

Reply:
Ohh, thanks. Will look at it tomorrow.

Reply:
The file does not matter. It's just where we write the downloaded data to and which we later delete. But I'll change it anyways to have consistent with the downloaded file and the folder name.